mail_auto_follower_notify/tests/test_auto_follower_notify.py

```diff
@@ -12,17 +13,24 @@ def setUp(self):
             'name': 'Test Record',
         })
         self.user = self.env['res.users'].create({
-            'name': 'Test user',
-            'login': 'test_login',
-        })
+             'name': 'Test user',
+             'login': 'test_login',
+             'password': 'demo',
+             'email': 'test@yourcompany.com',
+            })
 
+    @at_install(False)
+    @post_install(True)
```

Comment:
@hbrunn FYI finally the best approach was to run the test after the installation.

Reply:
good one!